android/src/main/java/com/swmansion/rnscreens/Screen.java

```diff
@@ -151,7 +152,22 @@ public void setTransitioning(boolean transitioning) {
       return;
     }
     mTransitioning = transitioning;
-    super.setLayerType(transitioning ? View.LAYER_TYPE_HARDWARE : View.LAYER_TYPE_NONE, null);
+    boolean isWebViewInScreen = findWebView(this);
+    super.setLayerType(transitioning && !isWebViewInScreen ? View.LAYER_TYPE_HARDWARE : View.LAYER_TYPE_NONE, null);
+  }
+
+  private boolean findWebView(ViewGroup viewGroup) {
```

Comment:
`find` should probably give back the `WebView` itself, not a `boolean` value

```suggestion
    boolean isWebViewInScreen = hasWebView(this);
    super.setLayerType(transitioning && !isWebViewInScreen ? View.LAYER_TYPE_HARDWARE : View.LAYER_TYPE_NONE, null);
  }

  private boolean hasWebView(ViewGroup viewGroup) {
```